src/ClientHandle.cpp

```diff
@@ -926,6 +926,12 @@ void cClientHandle::HandleBeaconSelection(int a_PrimaryEffect, int a_SecondaryEf
 
 void cClientHandle::HandleCommandBlockBlockChange(int a_BlockX, int a_BlockY, int a_BlockZ, const AString & a_NewCommand)
 {
+	if (a_NewCommand.empty())
+	{
+		Kick("Command block string unexpectedly empty - hacked client?");
+		return;
+	}
+
```

Comment:
I still think an empty command should be allowed. There's no reason not to, is there? Empty commands still need to be supported because that's the default state of a newly created command block. If Mojang ever decides to support empty commands, this would get in our way.